pandas/io/json.py

```diff
@@ -726,8 +726,8 @@ def nested_to_record(ds, prefix="", level=0):
 def json_normalize(data, record_path=None, meta=None,
                    meta_prefix=None,
                    record_prefix=None,
-                   errors='raise'):
-
+                   errors='raise',
+                   sep='.'):
```

Comment:
@jreback Comment on the previous pull request indicated that I should put this last in order to not break any current code that uses positional arguments.